src/components/Footer.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+const Footer = () => {
+    return (
+        <div className="text-center p-3 bg-primary text-white">
```

Comment:
Change `bg-primary` to `bg-background` so shadcn can recognise light and dark mode